### PR TITLE
 readme file updated

### PR DESCRIPTION
**Pull request by larathordar**
#### Description
Changes proposed in this pull request:

* gems added to readme
* instructions to run tests added 
#### What I have learned working on this feature:
to write a proper readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ Rails 5.2
 $ rvm gemset create rails_5_2
 $ rvm gemset use rails_5_2
 $ gem install rails --pre
+$ gem devise
+$ gem cucumber-rails
+$ gem 'rspec-rails'
+$ gem 'shoulda-matchers'
+$ gem 'cucumber-rails', require: false
+$ gem 'launchy'
```
Comment (tochman) [inline README.md]: I'm not sure if listing gems used in the app in conjunction with the commands used to create the app is a good move. Might be a bit confusing to the reader.

File: README.md [hunk 2]
```diff
@@ -36,7 +42,9 @@ End with an example of getting some data out of the system or using it for a lit
 
 ## Running the tests
 
-Explain how to run the automated tests for this system
+Navigate to your terminal.
+Make sure to be in the Dream team repository
+Run ```rake``` in the terminal.
```
Comment (tochman) [inline README.md]: If you want to add inline code, one back-tick is enough.

**Comment by jyuw** [comment]
Looks good!

File: README.md [hunk 3]
```diff
@@ -59,26 +62,26 @@ Give an example
 Add additional notes about how to deploy this on a live system
 
 ## Built With
-
 * Ruby on Rails 5.2 - The web framework used
 
-
-## Contributing
+# Gems for testing
```
Comment (magnus-thor) [inline README.md]: Be mindful of the header sizes.

**Review by jyuw — DISMISSED** [review]
Thanks for the updates

File: README.md [hunk 4]
```diff
@@ -59,26 +62,26 @@ Give an example
 Add additional notes about how to deploy this on a live system
 
 ## Built With
-
 * Ruby on Rails 5.2 - The web framework used
 
-
-## Contributing
+* Gems for testing
```
Comment (magnus-thor) [inline README.md]: Having this as a header was perfectly fine, and actually preferred that, it just that the header size was bigger then the `Built with` header,  and the `gems for testing` header should have been smaller then `Built with`

**Review by magnus-thor — APPROVED** [review]
Looks good :)

**Comment by holgertidemand** [comment]
looks good

**Review by jyuw — APPROVED** [review]

**Pull request merged**